service-map/src/main/java/fi/nls/oskari/map/style/VectorStyleMapper.java

```diff
@@ -36,7 +36,7 @@ public interface VectorStyleMapper {
     @Select("SELECT * FROM oskari_maplayer_style WHERE layer_id = #{layerId} AND (creator IS NULL OR creator=#{user})")
     List<VectorStyle> getStyles(@Param("user") long user, @Param("layerId") int layerId);
 
-    @Delete("DELETE FROM oskari_maplayer_style WHERE id = #{id} RETURNING id")
+    @Select("DELETE FROM oskari_maplayer_style WHERE id = #{id} RETURNING id")
```

Comment:
I don't think we really need the "returning id" here. Is that why Select-annotation is used instead of Delete? I would prefer the Delete and without the returning id if that is the reason for using Select?

Reply:
You are right that id isn't used/needed because handler returns true if everything goes right. Changed delete to select because mapper didn't delete row from db. Not sure if returning id causes this or how @Delete should work at all. Checked from Announcements mapper and there select is used for delete.

Reply:
Missing commit caused that delete statement/annotation didn't work. Maybe MyBatis notices that @Select delete updates data in db so it auto-commits.